pdh_check/datadog_checks/pdh_check/data/conf.yaml.example

```diff
@@ -2,6 +2,22 @@ init_config:
 
 instances:
 
+    ## @param host - string - required
+    ## Host that the Datadog PDH check connects to.
+    ## "." means the current host
+    #
+    # host: <HOST>
+
+    ## @param username - string - optional
+    ## Username from the credentials needed to connect to the host.
+    #
+    # username: <USER_NAME>
+
+    ## @param password - string - optional
+    ## Password from the credentials needed to connect to the host.
+    #
+    # password: <PASSWORD>
+
     ## The following example fetches the number of processes and users:
```

Comment:
Should we deprecate countersetname and metrics in favor of additional_metrics? Does it make sense to have additional_metrics in this integration even if its supported by the base class?

Reply:
I agree that it doesn't make a lot of sense to have both. I'd be more inclined to stay backward compatible and keep `additional_metrics` hidden from the example configuration file.